README.md

```diff
@@ -854,12 +856,12 @@ _When applying the concept of a Decider to event sourcing, the consistency requi
 #### The ingredients
 
 With Deciders in general, and Equinox in particular, the following elements are involved:
-- a `State` type, on which we base decisions, which can be updated by the consequences of a decision
-- a `decide` function, which is presented a `State`, and returns a decision, which we use to update the State, if, and only if there is no concurrency conflict when applying them
-- an `initial` State, from which we start
-- an `Event` type (think a Discriminated union)
-- the Events are maintained as an ordered list of events (could be an in memory array, a stream in ESDB, documents with a Tip as with `Equinox.CosmosStore`)
-- the State is established by [`fold`](https://en.wikipedia.org/wiki/Fold_(higher-order_function))ing the sequence of `Event`s, starting from an `initial` state
+- a `State` type, on which decisions can be based. This can be updated as a consequence of a decision. e.g. the item identifiers in a cart and the associated quantities
```

Comment:
```suggestion
- a `State` type, on which decisions can be based. This can be updated as a consequence of a decision, e.g. the item identifiers in a cart and the associated quantities
```